backend/src/main/java/de/kaiheinrich/projectplace/service/ProfileService.java

```diff
@@ -12,15 +20,48 @@
 @Service
 public class ProfileService {
 
+    @Value("${aws.access.key}")
+    private String accessKey;
+
+    @Value("${aws.secret.key}")
+    private String secretKey;
+
     private final ProfileMongoDb profileDb;
 
+
     @Autowired
     public ProfileService(ProfileMongoDb profileDb) {
         this.profileDb = profileDb;
     }
 
     public List<Profile> getProfiles() {
-        return profileDb.findAll();
+
+        Regions clientRegion = Regions.EU_CENTRAL_1;
+        BasicAWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+        String bucketName = "kais-super-bucket";
```

Comment:
That's the best bucket :D Maybe put the bucket name into an environment variable to make it configurable without recompiling?